platform/features/timeline/bundle.js

```diff
@@ -182,6 +182,16 @@ define([
                                 "capacity"
                             ],
                             "pattern": "^-?\\d+(\\.\\d*)?$"
+                        },
+                        {
+                            "name": "Battery Starting SOC",
+                            "control": "textfield",
+                            "required": false,
+                            "conversion": "number",
+                            "property": [
+                                "startingSOC"
+                            ],
+                            "pattern": "^-?\\d+(\\.\\d*)?$"
```

Comment:
I'm not certain, but I think users may prefer to enter this as a percentage; I've [asked here](https://github.com/nasa/openmct/issues/1185#issuecomment-248082245). Percentages are how I'm used to seeing state-of-charge visualized, at least. If so, we can use the `pattern` to keep values in the 0-100 range (allowing for decimals)

Otherwise, we should decide how to handle the case where `startingSOC > capacity`. Ideally this should be handled by validation (a `validate` function for this form row) but that may require additional changes to form handling (in platform/forms) to get access to the current `capacity` value from that `validate` function.